### DISCUSSION TITLE
Request for help

### DISCUSSION THREAD

**Discussion by rxndxm** [TomGrozev/arangox_ecto#36] [Q&A]
I'm looking for someone familiar with this library to answer some questions over Zoom or Telegram and possibly work with me on some small contributions (paid if needed!). @TomGrozev are you open to this, or have anyone you'd recommend?

I really want to use this library and it's *almost* what I need, but I'm in need of a bit of guidance 🙏 

Thanks!!

**Comment by TomGrozev** [comment]
I am unavailable to do any one-on-one help but can help with any questions you may have.
